src/types/api.ts

```diff
@@ -1,5 +1,9 @@
 import {PullRequestReviewState} from './gql-types';
 
+export type DashSecrets = {
+  GITHUB_CLIENT_ID: string; GITHUB_CLIENT_SECRET: string;
+};
```

Comment:
I don't think this should live here, `api.ts` is meant to document the types necessary to share between the server and client.